glusterd2/pmap/registry_test.go

```diff
@@ -14,21 +14,17 @@ func TestRegistry(t *testing.T) {
 	assert := require.New(t)
 
 	r := &pmapRegistry{
-		basePort:    1000,
-		bricks:      make(map[string]int),
-		conns:       make(map[net.Conn]int),
-		portLockFds: make(map[int]int),
+		Ports:  make(map[int]brickSet),
+		bricks: make(map[string]int),
+		conns:  make(map[net.Conn]int),
 	}
 
-	for i := 1000; i <= (r.basePort + 100); i++ {
-		r.ports[i].State = PortLeased
-	}
+	basePort := 49152
```

Comment:
How are you sure about this? Should this be 1024?

Reply:
This is arbitrary. It can be anything in the range 1024 to 65535.